common/flatpak-dir.c

```diff
@@ -6303,19 +6303,15 @@ flatpak_dir_update_remote_configuration (FlatpakDir   *self,
       {
         FlatpakSystemHelper *system_helper;
         g_autofree char *config_data = g_key_file_to_data (config, NULL, NULL);
-        g_autoptr(GVariant) gpg_data_v = NULL;
-
-        gpg_data_v = g_variant_ref_sink (g_variant_new_from_data (G_VARIANT_TYPE ("ay"), "", 0, TRUE, NULL, NULL));
 
         system_helper = flatpak_dir_get_system_helper (self);
         g_assert (system_helper != NULL);
 
-        g_debug ("Calling system helper: ConfigureRemote");
-        if (!flatpak_system_helper_call_configure_remote_sync (system_helper,
-                                                               0, remote,
-                                                               config_data,
-                                                               gpg_data_v,
-                                                               cancellable, error))
```

Comment:
We don't need the GPG data to update the remote's metadata (as it's reflected by this function `flatpak_dir_update_remote_configuration()` not receiving any gpg data as parameter) so there's no point on generating a dummy one and sending it over.